### PR TITLE
Basic concepts assignments problem 2

### PR DESCRIPTION
**Pull request by Haider0007**
Create a new string which is n copies of a given string where n is a non-negative integer.
     	A
	AA
	AAA
	AAAA
	AAAAA

### CODE REVIEW

**Comment by wqsaali** [comment]
@Haider0007 please add problem/question detail in description

File: problem2.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+puts "Enter Alphabet to print:"
+alphabet=gets.chomp!
+
+valid_num=false
```
Comment (wqsaali) [inline problem2.rb:4]:
```
input_number=Integer(gets.chomp!)
if input_number.negative?
  puts 'Wrong Input! Please Enter Again'
else
  1.upto(input_number) do |x|
    puts alphabet * x
  end
end

File: problem2.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+until valid_num==true
+  puts "Enter number of copies to create:"
+  num_copies=Integer(gets.chomp!)
+  
+  valid_num=num_copies>=0? true : false
+  puts "Wrong Input! Please Enter Again" unless valid_num
```
Comment (wqsaali) [inline problem2.rb]: if you have only string then use single quote.
double quote do two things. 
1. Interpolation
2. Save string.
That's why it is costly as compared to single quote

File: problem2.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+puts "Enter Alphabet to print:"
+alphabet=gets.chomp!
+
+valid_num=false
+until valid_num==true
+  puts "Enter number of copies to create:"
```
Comment (wqsaali) [inline problem2.rb]: use single quote for string

File: problem2.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  num_copies=Integer(gets.chomp!)
+  
+  valid_num=num_copies>=0? true : false
+  puts "Wrong Input! Please Enter Again" unless valid_num
+end  
+
+1.upto(num_copies) do |x|
+  puts alphabet * Integer(x)
```
Comment (wqsaali) [inline problem2.rb]: no need to explicity convert to Integer as x is already an integer

Instead of `x` variable name should be `index`

**Review by wqsaali — CHANGES_REQUESTED** [review]

File: problem2.rb (hunk 1 above)
Comment (Haider0007) [reply problem2.rb:4]: I have uploaded updated solution and 'until' is used in the solution for taking input from user till he enters some valid input.

**Pull request merged**